README.md

```diff
@@ -53,26 +49,65 @@ A user can add additional writers to each song.
 
 ![App Screenshots](./src/images/dtty-add-writer.gif)
 
+---
+
```

Comment:
I find instructions like this to be the standard style. A lot of the community use Yarn or npm, so I like to add directions for both. Expect that whoever is following these instructions has done this kind of thing before. Keep it to the point, and make it as easy as possible 😉